channeld/channeld.c

```diff
@@ -3234,9 +3235,12 @@ static void handle_funding_depth(struct peer *peer, const u8 *msg)
 				     peer->next_index[LOCAL],
 				     type_to_string(tmpctx, struct pubkey,
 						    &peer->next_local_per_commit));
-			msg = towire_funding_locked(NULL,
-						    &peer->channel_id,
-						    &peer->next_local_per_commit);
+			struct tlv_funding_locked_tlvs *tlvs =
+			    tlv_funding_locked_tlvs_new(tmpctx);
```

Comment:
BTW, NULL works the same as an empty TLV.  Though I suspect you want this allocation for a future patch...